apps/prairielearn/src/pages/administratorCourses/administratorCourses.js

```diff
@@ -19,8 +27,41 @@ router.get(
   '/',
   asyncHandler(async (req, res) => {
     res.locals.coursesRoot = config.coursesRoot;
-    const result = await sqldb.queryOneRowAsync(sql.select, []);
-    _.assign(res.locals, result.rows[0]);
+    const result = await sqldb.queryRow(
+      sql.select,
+      [],
+      z.object({
+        institutions: z.array(InstitutionSchema),
+        courses: z.array(
+          CourseSchema.extend({
+            institution: InstitutionSchema,
+          }),
+        ),
+        course_requests: z.array(
+          CourseRequestSchema.extend({
+            user_name: z.string(),
+            created_at: DateFromISOString.optional(),
+            approved_by: z.string().optional(),
+            approved_status: z
+              .enum(['pending', 'approved', 'denied', 'creating', 'failed'])
+              .optional(),
+            user_id: z.string().optional(),
+            jobs: z
+              .object({
+                start_date: DateFromISOString.optional(),
+                finish_date: DateFromISOString.optional(),
+                authn_user_id: IdSchema.optional(),
+                status: z.string().optional(),
+                id: IdSchema.optional(),
+                number: z.number().optional(),
+              })
+              .optional(),
+          }),
+        ),
+      }),
+    );
+    console.log(result);
+    _.assign(res.locals, result);
```

Comment:
Same comment from the other file.

Reply:
Removed the usage of lodash in commit [9dd4a64](https://github.com/PrairieLearn/PrairieLearn/pull/9238/commits/9dd4a647f1e05a13c5ced774f7b979c7d749fbf4).